ui/js/directives/treeherder/clonejobs.js

```diff
@@ -249,18 +262,17 @@ treeherder.directive('thCloneJobs', [
             jobStatus.key = getJobMapKey(job);
             jobStatus.value = job.job_type_symbol;
             jobStatus.title = getHoverText(job);
-
+            jobStatus.visibleClass = job.visible ? "filter-shown" : "";
             if (thResultStatus(job) === "runnable") {
                 jobStatus.buildername = job.ref_data_name;
-                jobBtn = $(runnableJobBtnInterpolator(jobStatus));
-
                 if (ThResultSetStore.isRunnableJobSelected($rootScope.repoName,
                                                            job.result_set_id,
                                                            jobStatus.buildername)) {
-                    jobBtn.addClass("runnable-job-btn-selected");
+                    jobStatus.runnableClass = "runnable-job-btn-selected";
                 }
+                jobBtn = runnableJobBtnInterpolator(jobStatus);
             } else {
-                jobBtn = $(jobBtnInterpolator(jobStatus));
+                jobBtn = jobBtnInterpolator(jobStatus);
             }
 
             // If the job is currently selected make sure to re-apply
```

Comment:
Just realized that this block of code needs to be updated to handle the fact that we no longer have a jQuery element to add the class to. You'll need to add the classes applied in `setSelectJobStyles` to the interpolated job element, then re-select the job once it exists. The first part should be relatively straightforward, the second might be tricky -- probably the right way of doing this is to make the resultset store model no longer keep a reference to the job element -- we should get that element ourselves where needed based purely on the id. 

If you want to try tackling this, you'll need to modify `ui/js/models/resultsets_store.js`, in particular the getSelectedJob and setSelectedJob to only keep a reference to the selected job's id, not el. You'll also need to add a jQuery selector inside clonejobs.js to get the element dynamically based on the id (where we were getting it via getSelectedJob before).